<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadIntegrator</class>
 <widget class="QMainWindow" name="DownloadIntegrator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FLiNG修改器下载器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <layout class="QHBoxLayout" name="searchLayout">
      <item>
       <widget class="QLineEdit" name="searchEdit">
        <property name="placeholderText">
         <string>搜索游戏...</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>搜索</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/resources.qrc">
          <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sortComboBox">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>最近更新</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按名称</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>下载次数</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="toolTip">
         <string>从网站获取最新修改器列表</string>
        </property>
        <property name="text">
         <string>最近更新</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/resources.qrc">
          <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="leftWidget" native="true">
       <layout class="QVBoxLayout" name="leftLayout">
        <item>
         <widget class="QTableWidget" name="modifierTable">
          <column>
           <property name="text">
            <string>游戏名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>更新日期</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>支持版本</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>选项数量</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rightWidget" native="true">
       <layout class="QVBoxLayout" name="rightLayout">
        <item>
         <widget class="QLabel" name="gameTitle">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="versionGroup">
          <property name="title">
           <string>版本信息</string>
          </property>
          <layout class="QVBoxLayout" name="versionLayout">
           <item>
            <widget class="QLabel" name="versionInfo">
             <property name="text">
              <string>游戏版本：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="optionsCount">
             <property name="text">
              <string>修改器选项：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lastUpdate">
             <property name="text">
              <string>最后更新：</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="downloadGroup">
          <property name="title">
           <string>下载选项</string>
          </property>
          <layout class="QVBoxLayout" name="downloadLayout">
           <item>
            <widget class="QRadioButton" name="autoUpdateVersion">
             <property name="text">
              <string>自动更新版本</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="standaloneVersion">
             <property name="text">
              <string>独立版本</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="versionSelect"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="modifierOptions">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>修改器功能选项列表...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="downloadProgress">
          <property name="visible">
           <bool>false</bool>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="buttonLayout">
          <item>
           <widget class="QPushButton" name="downloadButton">
            <property name="text">
             <string>下载</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="openFolderButton">
            <property name="text">
             <string>打开下载目录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="settingsButton">
            <property name="text">
             <string>设置</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSettings">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>